AdyenComponents/UPI/UPIComponent.swift

```diff
@@ -111,14 +118,47 @@ public final class UPIComponent: PaymentComponent,
     /// The  virtual payment address text input item.
     internal lazy var virtualPaymentAddressItem: FormTextInputItem = {
         let item = FormTextInputItem(style: configuration.style.textField)
-        item.title = Constants.virtualPaymentAddress
+        // TODO: Add localised string
+        item.title = "Enter UPI ID / VPA"
```

Comment:
Localization strings are not ready yet?

Reply:
They are. I am going to add them in separate PR today.